src/ipc/StoreMap.h

```diff
@@ -259,9 +259,14 @@ class StoreMap
     /// undoes partial update, unlocks, and cleans up
     void abortUpdating(Update &update);
 
-    /// only works on locked entries; returns nil unless the slice is readable
+    /// the caller must hold a lock on the entry
+    /// \returns nullptr unless the slice is readable
     const Anchor *peekAtReader(const sfileno fileno) const;
 
+    /// the caller must hold a lock on the entry
+    /// \returns nullptr unless the slice is writeable
+    const Anchor *peekAtWriter(const sfileno fileno) const;
+
     /// only works on locked entries; returns the corresponding Anchor
```

Comment:
Stale comment format. Please update the description to stay in sync with the other two peeking method descriptions.

Reply:
Done.